### DISCUSSION TITLE
Invalid translation [fr] - Addon "move costume up or down"

### DISCUSSION THREAD

**Discussion by wilhelmbismarck** [ScratchAddons/ScratchAddons#7844] [Translation] [answered]
🇫🇷 French translation contains two grammatical errors : 
- `move to top` translated `déplacé en haut` while it should be `déplacer en haut` (-> *use infinitive for a common action*)
- `move to bottom` translated `déplacé en bas` while it should be `déplacer en bas` (-> *same*)

Have a good day ;)

**Comment by WorldLanguages** [comment, marked as answer]
Fixed, thank you!

It will reach users after the next update.
